batcher/aligned-batcher/src/lib.rs

```diff
@@ -441,7 +441,7 @@ impl Batcher {
 
         if client_msg.verification_data.chain_id != self.chain_id {
             warn!(
-                "Received message with incorrect chain id: {}",
+                "Received message with incorrect chain id: {}", //This check does not save against "Holesky" and "HoleskyStage", since both are chain_id 17000
```

Comment:
Is there any other check that can cover the difference?

Reply:
yes, on line `459`: `This checks saves against "Holesky" and "HoleskyStage", since each one has a different payment service address`